test/extensions/filters/udp/udp_proxy/udp_proxy_filter_test.cc

```diff
@@ -392,16 +429,26 @@ cluster: fake_cluster
   recvDataFromDownstream("10.0.0.1:1000", "10.0.0.2:80", "hello");
   EXPECT_EQ(2, config_->stats().downstream_sess_total_.value());
   EXPECT_EQ(1, config_->stats().downstream_sess_active_.value());
+
+  filter_.reset();
+  EXPECT_EQ(access_log_data_.value(), "2 1");
 }
 
 // Verify downstream send and receive error handling.
 TEST_F(UdpProxyFilterTest, SendReceiveErrorHandling) {
   InSequence s;
 
-  setup(R"EOF(
+  const std::string access_log_format =
+      "%BYTES_SENT% %BYTES_RECEIVED% "
+      "%DOWNSTREAM_HEADER_BYTES_SENT% %DOWNSTREAM_HEADER_BYTES_RECEIVED% "
+      "%DOWNSTREAM_WIRE_BYTES_SENT% %DOWNSTREAM_WIRE_BYTES_RECEIVED% "
+      "%UPSTREAM_HEADER_BYTES_SENT% %ROUTE_NAME%";
```

Comment:
Can you add some separated access log tests. It is difficult to check if all supported formats have been tested.